contrib/sui_config/sui.keystore

```diff
@@ -0,0 +1,3 @@
+[
```

Comment:
This is private key?

Reply:
yes. Its for the local e2e tests. This is the volume we import in the docker contrainer